target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -466,6 +466,9 @@ ramips_board_detect() {
 	*"SL-R7205"*)
 		name="sl-r7205"
 		;;
+	*"Telco Electronics X1")
```

Comment:
Since ramips uses DT-based device discovery, you don't need this hunk anymore. Please remove it.